tests/CMakeLists.txt

```diff
@@ -123,6 +123,12 @@ if(QUDA_DIRAC_STAGGERED)
   quda_checkbuildtest(staggered_eigensolve_test QUDA_BUILD_ALL_TESTS)
   install(TARGETS staggered_eigensolve_test ${QUDA_EXCLUDE_FROM_INSTALL} DESTINATION ${CMAKE_INSTALL_BINDIR})
 
+  if(QUDA_CONTRACT)
+    add_executable(contractFT_test contractFT_test.cpp)
```

Comment:
We need a ctest instance that tests `contractFT_test`

Reply:
resolved in 4d43b5d46947a1a1b760b33ee5859d5d12bffc0a